docs/UserGuide.md

```diff
@@ -134,35 +184,51 @@ Format: `edit INDEX [c/COMPANY] [r/ROLE] [s/STAGE] [d/DATETIME] [e/COMPANY_EMAIL
 * You can remove all the internship’s tags by typing `t/` without
     specifying any tags after it.
 
+<div markdown="block" class="alert alert-success">
+**:bulb: Tip:** You can edit any number of attributes with the edit command!
+</div>
+
 Examples:
 * `edit 1 s/Behavioural Interview e/hr@meta.com` Edits the stage and email address of the first internship to be `Behavioural Interview` and `hr@meta.com` respectively.
 * `edit 2 l/golang t/` Adds `golang` as the only language tag for the second internship and clears all existing tags.
 
-### 4.3. Listing all existing internship applications: 
+### 6.3. Listing all existing internship applications: 
 
 Lists all of your internship applications in WorkBook.
 
 Format: `list`
 
-### 4.4. Finding your internships by company, stage or role name:
+### 6.4. Finding your internships by company, stage or role name:
 
-Finds internships whose company, role or stage names contain any of the respective keywords.
+If you wanted to view all your internship applications corresponding to a particular company, stage or role,
+then this command is for you! <br>
+
+It can find internships whose company, role or stage names contain any of the respective keywords.
 
 Format: `find c/COMPANY | r/ROLE | s/STAGE`
 
 * Displays a list of internships that match all the keywords.
-* Case does not matter e.g. `meta` will match `Meta`.
+
+<div markdown="block" class="alert alert-success">
+**:bulb: Tip:** <br>
 * Only full words will be matched e.g. `met` will not match `Meta`.
 * Order of the keywords does not matter e.g. `Jane Street` will match `Street Jane`.
+</div>
+
+<div markdown="block" class="alert alert-warning">
+**:exclamation: Caution**
+* Only full words will be matched e.g. `met` will not match `Meta`.
 * Exactly one attribute can be searched for either Company, Role or Stage.
+</div>
 
 
 Examples:
 * `find r/Engineer` returns `Software Engineer` and `Backend Engineer`.
 * `find s/Interview` returns `Technical Interview` and `Behavioural Interview`.
 
-### 4.5. Deleting your internship application:
+### 6.5. Deleting your internship application:
 
+If you wanted to remove an internship application then this command,<br>
 Deletes the specified internship application from WorkBook.
```

Comment:
not a fan of this line break here and then the capital letter on the next line

<img width="649" alt="Screenshot 2022-11-01 at 2 47 46 PM" src="https://user-images.githubusercontent.com/77665182/199175408-9f7c288d-9d6d-4961-8467-c11182c5a854.png">